sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/models/TableErrorCode.java

```diff
@@ -8,7 +8,65 @@
 /**
  * Defines values for error codes returned from the Tables service.
  */
-public class TableErrorCode extends ExpandableStringEnum<TableErrorCode> {
+public final class TableErrorCode extends ExpandableStringEnum<TableErrorCode> {
+    /**
+     * Static value {@code InvalidTableName}.
+     */
+    public static final TableErrorCode INVALID_TABLE_NAME = fromString("InvalidTableName");
```

Comment:
Where are these error codes documented? Can we add a link to it in the JavaDoc? Also, is this something that we can include in the swagger instead of manually enumerating each error code coming from the service?

Reply:
Error codes can be found here:

- [Common Storage error codes](https://docs.microsoft.com/en-us/rest/api/storageservices/common-rest-api-error-codes)
- [Tables Specific codes](https://docs.microsoft.com/en-us/rest/api/storageservices/table-service-error-codes)

Reply:
Would the link be documented on the Exception and Error classes or somewhere else?

Reply:
I think we can ask the service team to include them in the swagger, what do you think @annatisch?